src/Compilers/Core/Portable/Symbols/NullableFlowState.cs

```diff
@@ -23,4 +23,20 @@ public enum NullableFlowState : byte
         /// </summary>
         MaybeNull
     }
+
+    internal static class NullableFlowStateExtensions
+    {
+        public static NullableAnnotation ToAnnotation(this NullableFlowState nullableFlowState)
```

Comment:
Please add a comment to this talking about how it can produce annotation states that are impossible, such as an unconstrained T that is annotated.

Reply:
> Please add a comment to this talking about how it can produce annotation states that are impossible, such as an unconstrained T that is annotated.

Isn't that common knowledge that the model is capable of representing "impossible" (from the language point of view) situations and, in fact, does that very often? 

---
In reply to: [357762538](https://github.com/dotnet/roslyn/pull/40338#discussion_r357762538) [](ancestors = 357762538)